artichoke-backend/src/extn/core/string/mruby.rs

```diff
@@ -439,6 +441,17 @@ unsafe extern "C" fn string_empty(mrb: *mut sys::mrb_state, slf: sys::mrb_value)
     }
 }
 
+unsafe extern "C" fn string_end_with(mrb: *mut sys::mrb_state, slf: sys::mrb_value) -> sys::mrb_value {
+    let suffixes = mrb_get_args!(mrb, *args);
+    unwrap_interpreter!(mrb, to => guard);
+    let value = Value::from(slf);
+    let suffixes = suffixes.iter().map(|&other| Value::from(other));
+    match trampoline::end_with(&mut guard, value, suffixes) {
```

Comment:
I'm not sure what putting this expression in the match means for lifetimes and things. Because the Err branch unwinds with a longjmp, I'd feel more comfortable if the result of the trampoline was assigned to a local binding like it is in the other mruby entry points.